.github/scripts/update-featured-repo.js

```
@@ -0,0 +1,126 @@
+      stars: repoData.stargazers_count,
+      isFork: repoData.fork
+    };
+  } catch (error) {
+    console.error('Error fetching repository:', error);
+    return null;
+  }
+}
+
+function updateReadme(repo) {
+  try {
+    let readme = fs.readFileSync(README_PATH, 'utf8');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'readme' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)